### DISCUSSION TITLE
Error when setting up git integration

### DISCUSSION THREAD

**Discussion by nooblyf** [vercel/vercel#4623] [Help] [answered]
I've read the docs for linking github account to vercel but i'm getting this message:

This installation is initialized by another Vercel account.
To complete the process, login as a different user or share this link to the owner:

![Annotation](https://user-images.githubusercontent.com/54323745/84351230-8de37080-abd8-11ea-9cc5-5e308669ae4d.jpg)

**Comment by nooblyf** [comment, marked as answer]
There was a hyphen in my GitHub username earlier, after removing it I was able to connect, idk if that was the problem.

**Reply by styfle** [reply]
Hi @nooblyf 

It sounds like you renamed your GitHub username and the Vercel Git Integration thought it was a different GitHub user.

If you are still having issues logging in, send an email to support@vercel.com and we'll dig deeper to find out what is going on here, thanks!
